incremental-font-transfer/src/patch_group.rs

```diff
@@ -0,0 +1,1253 @@
+use read_fonts::{tables::ift::CompatibilityId, FontRef, ReadError, TableProvider};
+use std::collections::{BTreeMap, HashMap};
+
+use crate::{
+    font_patch::{IncrementalFontPatchBase, PatchingError},
+    patchmap::{intersecting_patches, IftTableTag, PatchEncoding, PatchUri, SubsetDefinition},
+};
+
+/// A group of patches derived from a single IFT font which can be applied simultaneously
+/// to that font. Patches are initially missing data which must be fetched and supplied to
+/// the group before it can be applied to the font.
```

Comment:
doc comments sort of work like git commits; you want a short first line, then whitespace, then you can elaborate.

The first line is used as a summary when the docs are rendered.